### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by LiamConnors**
We have these links on the reference docs

image

That go here: https://plotly.com/python-api-reference/generated/generated/plotly.graph_objects.Figure.html

But the names in the built api docs are different.

image

https://github.com/plotly/plotly.py-docs/tree/gh-pages/generated/generated

We are renaming everything else to `graph_objects` except these. 

This PR should fix the links, but I need to investigate more why these are in a different directory and if they should be included.

### CODE REVIEW

**Comment by LiamConnors** [comment]
Full build failing: https://github.com/plotly/plotly.py/issues/4619
Doesn't seem to be related to this. Has been failing on other PRs

File: doc/apidoc/Makefile [hunk 1]
```diff
@@ -36,7 +36,7 @@ help:
 	# Remove files which were added only for docstring generation
 	rm ../../packages/python/plotly/plotly/colors/diverging.py ../../packages/python/plotly/plotly/colors/sequential.py ../../packages/python/plotly/plotly/colors/qualitative.py ../../packages/python/plotly/plotly/colors/cyclical.py ../../packages/python/plotly/plotly/colors/colorbrewer.py ../../packages/python/plotly/plotly/colors/carto.py ../../packages/python/plotly/plotly/colors/cmocean.py
 	rm ../../packages/python/plotly/plotly/express/colors/diverging.py ../../packages/python/plotly/plotly/express/colors/sequential.py ../../packages/python/plotly/plotly/express/colors/qualitative.py ../../packages/python/plotly/plotly/express/colors/cyclical.py ../../packages/python/plotly/plotly/express/colors/colorbrewer.py ../../packages/python/plotly/plotly/express/colors/carto.py ../../packages/python/plotly/plotly/express/colors/cmocean.py
-	rename 's/graph_objs/graph_objects/' _build/html/*.html _build/html/generated/*.html
+	rename 's/graph_objs/graph_objects/' _build/html/*.html _build/html/generated/*.html _build/html/generated/generated/*.html
```
Comment (gvwilson) [inline doc/apidoc/Makefile:39]: I don't understand why `generated/generated` but I trust you :-)

**Review by gvwilson — APPROVED** [review]
Approved pending your decision on whether this can be merged despite #4619.

**Comment by LiamConnors** [comment]
> Approved pending your decision on whether this can be merged despite #4619.

I'll try and fix that first as we'll need to fix it to do the next Plotly.py release

**Pull request merged**